src/Configuration/Cache/IlluminateCacheAdapter.php

```diff
@@ -54,7 +54,11 @@ protected function doSave($id, $data, $lifeTime = false)
             return $this->cache->forever($id, $data);
         }
 
-        return $this->cache->put($id, $data, $lifeTime);
+        // Laravel cache system accept expire times in minutes 
+        // while Doctrine accept it in seconds, so we need to convert $lifeTime to minutes
+        // before passing it to Laravel cache repository
+        $lifeTime_minutes = $lifeTime/60;
```

Comment:
Sure, 
I've also run php-cs-fixer but it return this error

`PHP Fatal error:  Uncaught Error: Class 'Symfony\CS\Finder\DefaultFinder' not found in C:\Users\alessandro\Desktop\orm\.php_cs:3
Stack trace:
#0 C:\Users\alessandro\AppData\Roaming\Composer\vendor\friendsofphp\php-cs-fixer\src\Console\ConfigurationResolver.php(213): include()
Fatal error: Uncaught Error: Class 'Symfony\CS\Finder\DefaultFinder' not found in C:\Users\alessandro\Desktop\orm\.php_cs:3
#1 C:\Users\alessandro\AppData\Roaming\Composer\vendor\friendsofphp\php-cs-fixer\src\Console\ConfigurationResolver.php(539): PhpCsFixer\Console\ConfigurationResolver->getConfig()
Stack trace:
#2 C:\Users\alessandro\AppData\Roaming\Composer\vendor\friendsofphp\php-cs-fixer\src\Console\ConfigurationResolver.php(389): PhpCsFixer\Console\ConfigurationResolver->getFormat()
#0 C:\Users\alessandro\AppData\Roaming\Composer\vendor\friendsofphp\php-cs-fixer\src\Console\ConfigurationResolver.php(213): include()
#3 C:\Users\alessandro\AppData\Roaming\Composer\vendor\friendsofphp\php-cs-fixer\src\Console\Command\FixCommand.php(140): PhpCsFixer\Console\ConfigurationResolver->getReporter()
#1 C:\Users\alessandro\AppData\Roaming\Composer\vendor\friendsofphp\php-cs-fixer\src\Console\ConfigurationResolver.php(539): PhpCsFixer\Console\ConfigurationResolver->getConfig()
#4 C:\Users\alessandro\AppData\Roaming\Composer\vendor\symfony\console\Command\Command.php(255): PhpCsFixer\Console\Command\FixCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\O`

(I'm on windows10, interpreter php7.1, php-cs-fixer 2.1.2)

Any idea on what could be the problem?